tests/cases/ssbo_with_graphics_pipeline.amber

```diff
@@ -12,6 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+[require]
+vertexPipelineStoresAndAtomics
```

Comment:
Can we move this to a different test since this one has a different purpose?

Reply:
Is it possible to add a negative test for this? Do we know if there is a feature that isn't supported anywhere?

Reply:
This test works without this requirement but just reports an validation error when enabling the validation layer.
Later, I will add some positive/negative tests that are suitable for checking required features.